docker-compose.yml

```diff
@@ -18,29 +18,29 @@ services:
     build:
       context: .
       dockerfile: Dockerfile
+    ports:
+      - ${PORT}:${PORT}
```

Comment:
As per the [docker-compose docs](https://docs.docker.com/compose/compose-file/#ports), please use a string here instead of direct numbers.